[Infra + Logs UI] Add source configuration ui to the docs (#33535)
elastic/kibana: docs/logs/index.asciidoc

[float]
== Add data

Kibana provides step-by-step instructions to help you add log data. The
{infra-guide}[Infrastructure Monitoring Guide] is a good source for more
detailed information and instructions.

[float]
== Configure data sources 

> By default the Logs UI uses the `filebeat-*` index pattern to query the data.

Reviewer: ```suggestion
By default, the Logs UI uses the `filebeat-*` index pattern to query the data.
```

> If your logs are located in a different set of indices or use a different

Reviewer: ```suggestion
If your logs are located in a different set of indices, or use a different
```

timestamp field, you can adjust the source configuration via the user interface
or the {kib} configuration file.

> NOTE: The Logs and Infrastructure UI share a common data source definition in

Reviewer: ```suggestion
NOTE: Logs and Infrastructure share a common data source definition in
```

each space. Changes in one of them can influence the data displayed in the
other.

[float]

> === Configure data sources in the user interface

Reviewer: ```suggestion
=== Configure data sources in Kibana
```

> The source configuration UI can be accessed via the corresponding button in the

Reviewer: Same comment here as `infrastructure/index.asciidoc`

toolbar.

[role="screenshot"]
image::logs/images/logs-configure-source.png[Configure Logs UI source button in Kibana]

This opens the source configuration fly-out dialog, in which the following
configuration items can be inspected and adjusted:

* *Name*: The name of the source configuration.
* *Indices*: The patterns of the elasticsearch indices to read metrics and logs
  from.
* *Fields*: The names of particular fields in the indices that need to be known
  to the Infrastructure and Logs UIs in order to query and interpret the data
  correctly.

[role="screenshot"]
image::logs/images/logs-configure-source-dialog.png[Configure logs UI source dialog in Kibana]

TIP: If <<xpack-spaces>> are enabled in your Kibana instance, any configuration

> changes performed via the user interface are specific to that space. You can

Reviewer: ```suggestion
changes performed via Configure source are specific to that space. You can
```

therefore easily make different subsets of the data available by creating
multiple spaces with different data source configurations.

[float]

> === Configure data sources in the configuration file

Reviewer: ```suggestion
=== Configuration file
```

[float]
== Add data

Kibana provides step-by-step instructions to help you add log data. The
{infra-guide}[Infrastructure Monitoring Guide] is a good source for more
detailed information and instructions.

[float]
== Configure data sources 

The `filebeat-*` index pattern is used to query data by default.
If your logs are located in a different set of indices, or use a different
timestamp field, you can adjust the source configuration via the user interface
or the {kib} configuration file.

NOTE: Logs and Infrastructure share a common data source definition in
each space. Changes in one of them can influence the data displayed in the
other.

[float]
=== Configure source

> The source configuration UI can be accessed via the corresponding

Reviewer: ```suggestion
Configure source can be accessed via the corresponding
```
